src/server_main.cc

```diff
@@ -35,7 +35,8 @@ int main(int argc, char** argv) {
      "Buffer cache time of the underlying TF buffer in seconds.")
     ("max_timeout", po::value<double>(),
      "Requests with lookup timeouts (seconds) above this will be blocked.")
-    ("debug", "Enables debugging features.")
+    ("service", "Advertise the view_frames service.")
+    ("debug", "Advertise the view_frames service (same as --service).")
```

Comment:
I think `frames_service` is better, just `--service` is a bit odd considering that we publish other services also without this option.
Also, the service is actually called `tf2_frames` instead of `view_frames`, the tf2_ros docs are outdated (I can make a PR to geometry2 to fix that...)